awx/ui/src/screens/Inventory/SmartInventoryHosts/SmartInventoryHostList.js

```diff
@@ -91,6 +99,7 @@ function SmartInventoryHostList({ inventory }) {
                     adHocItems={selected}
                     hasListItems={count > 0}
                     onLaunchLoading={setIsAdHocLaunchLoading}
+                    moduleOptions={moduleOptions}
```

Comment:
👍 Verified that adding this prop resolves the issue. Thanks for defining the PropTypes in AdHocCommands. Nice work!